index.html

```diff
@@ -7,19 +7,31 @@
     <link rel="preconnect" href="https://fonts.googleapis.com">
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
     <link href="https://fonts.googleapis.com/css2?family=Comforter&family=Noto+Sans+JP:wght@500&family=Roboto+Condensed&family=Roboto:wght@100&family=Sedgwick+Ave&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Comforter&family=Crete+Round&family=Noto+Sans+JP:wght@500&family=Roboto+Condensed&family=Roboto:wght@100&family=Sedgwick+Ave&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Comforter&family=Crete+Round&family=Noto+Sans+JP:wght@500&family=Poppins:wght@100&family=Roboto+Condensed&family=Roboto:wght@100;300&family=Sedgwick+Ave&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="style.css">
     <title>Portfolio</title>
 </head>
 <body>
-    <header class="top">
-        <div class="status">
-            <div>9:41</div>
-            <img src="images/Status_Bars/Group4.png" alt="status icons taken from the figma template">
+    <div class="container">
+        <header class="top">
+            <div class="status">
+                <div>9:41</div>
+                <img src="images/Status_Bars/Group4.png" alt="status icons taken from the figma template">
+            </div>
+            <div class="menu">
+                <div>John Doe</div>
+                <img src="images/Tertiary/Icons/Union.png" alt="menu bar. Icon taken from the figma template">
+            </div>
+        </header>
+        <div class="headline">
+            <div class="title">Hey There. I'm Oscar</div>
```

Comment:
Since this is the main heading of the page, Kindly wrap this with the ` h1 ` tag instead of ` div `. The site should only contain exactly one ` h1 ` tag